src/app-logic/constants.js

```diff
@@ -110,8 +110,7 @@ export const PROFILER_SERVER_ORIGIN = 'https://api.profiler.firefox.com';
 // SYMBOL_SERVER_URL
 // -----------------
 // Can be overridden with the URL parameter `symbolServer=SERVERURL`.
-// You can change this to the staging server `https://symbolication.stage.mozaws.net`,
-// or run a local symbol server (for example using profiler-symbol-server [1])
```

Comment:
Further, the `SYMBOL_SERVER_URL` constant below should be renamed to `SYMBOLICATION_SERVICE_URL` since the symbol server is a different thing.

Reply:
Yeah, looks like we got used to misname this :-)